IBPSA/Resources/Scripts/Dymola/Fluid/Actuators/Dampers/Validation/PressureIndependent.mos

```diff
@@ -0,0 +1,22 @@
+simulateModel("IBPSA.Fluid.Actuators.Dampers.Validation.PressureIndependent",
+    method="cvode", tolerance=1e-6, stopTime=1.0, resultFile="PressureIndependent");
+createPlot(id = 1,
+ position = {73, 9, 598, 390},
+ y = {"damPreInd.m_flow", "damExp.m_flow", "damExpPI.m_flow"},
+ range = {0.15, 0.6, 11.0, 7.0},
+ autoscale = true,
+ autoerase = true,
+ autoreplot = true,
+ grid = true,
+ color = true);
+createPlot(id=1, y={"damPreInd.y", "damPreInd.y_actual"},
+ position={73, 9, 598, 390},
+ autoscale = true,
+ autoerase = true,
+ autoreplot = true,
+ grid = true,
+ color = true,
+ subPlot=2);
+plotExpression(apply(PressureIndependent[end].senRelPre.p_rel/PressureIndependent[end].dp_nominal),
```

Comment:
Does it work to use `[end]` in these scripts? Might want to reconsider this formulation if it was not intentional.

Reply:
It works but it also works without so I removed it.